### ISSUE TITLE
Show keyboard focus

### ISSUE THREAD

**Issue by fieldOfView** [Ultimaker/Cura#172]
Pressing (shift) tab allows the user to step through the Print Settings. For numeric values this is show by a cursor in the input field, but for other types of values the keyboard focus is not so clear.

Section headers can receive keyboard focus (and respond to the spacebar to expand/collapse), but this keyboard focus is not visible.

When a checkbox gets the focus a focus "ring" is drawn next to a checkbox. This looks somewhat strange.

When a select box gas the keyboard focus, it does not show any different than when it has no focus.

Best solution would probably be to change the focused control style to the hover style for all control types.

**Comment by nallath** [comment]
I've implemented it as you suggested. It might be that we need to update the focus color as it's fairly subtle now. 

I've also added that the category headers are no longer in the 'tab' list, which should also give a better experience.
